bip-0325.mediawiki

```diff
@@ -60,8 +60,10 @@ The "to_sign" transaction is:
     vin[0].prevout.hash = to_spend.txid
     vin[0].prevout.n = 0
     vin[0].nSequence = 0
+    vin[0].sigScript = [ signet_challenge first data push, if any ]
+    vin[0].scriptWitness = [ signet_challenge second data push, if any ]
```

Comment:
These are from the signet solution, not the signet challenge, and there's only a single data push of the serialization of the scriptSig and and scriptWitness concatenated.

Reply:
Doh, I meant the signet challenge.

There's a single push, of the 1-2 pushes of sigScript followed by optional scriptWitness, right? That's what I'm referring to.

Reply:
Gah, I meant the solution, not challenge.